test/src/mbgl/test/conversion_stubs.hpp

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <mbgl/style/conversion.hpp>
+#include <mbgl/util/feature.hpp>
+#include <mbgl/util/optional.hpp>
+#include <mbgl/util/variant.hpp>
+
+#include <string>
+#include <map>
+
+namespace mbgl {
+namespace style {
+namespace conversion {
+
+class Holder;
```

Comment:
Can we rename `Holder` to something like `Value`? Usually on mbgl `Holder` is a temporary variable used to prevent something like a refcounted pointer to go out of scope.